Sources/PostgresNIO/Connection/PostgresConnection.swift

```diff
@@ -77,13 +77,23 @@ public final class PostgresConnection {
             ///
             /// - Default: 5432
             public var port: Int
-
-            public init(host: String, port: Int = 5432) {
+            
+            /// Require connection to provide `BackendKeyData`
+            ///
+            /// - Default: true
+            public var requireBackendKeyData: Bool
+            
+            public init(
+                host: String,
+                port: Int = 5432,
+                requireBackendKeyData: Bool = true
+            ) {
                 self.host = host
                 self.port = port
+                self.requireBackendKeyData = requireBackendKeyData
```

Comment:
```suggestion
            public init(host: String, port: Int = 5432) {
                self.host = host
                self.port = port
                self.requireBackendKeyData = true
```

Sadly the proposed change is an API breaking change, because of this...

Existing API:
```swift
struct Foo {
    init(bar: String) {}

    static func test() {
        let make = Foo.init(bar:)
        let foo = make("hello")
        // ...
    }
}
```

Adding a defaulted value breaks this code however:
```swift
struct Foo {
    init(bar: String, test: String = "foo") {}

    static func test() {
        let make = Foo.init(bar:) // <--- ❌ compiler error
        let foo = make("hello")
        // ...
    }
}
```

Reply:
Hadn't thought of this scenario! Will address along with the other comments. Thank you.